subsys/nrf_security/Kconfig.tls

```diff
@@ -256,6 +259,14 @@ config MBEDTLS_SSL_KEEP_PEER_CERTIFICATE
 	  reduces RAM usage.
 	  Corresponds to MBEDTLS_SSL_KEEP_PEER_CERTIFICATE in mbed TLS config file.
 
+config MBEDTLS_SSL_TLS1_3_COMPATIBILITY_MODE
```

Comment:
Move this one to be with the others, within the `if MBEDTLS_TLS_VERSION_1_3`?

Reply:
Yeah no idea why it was alone

Reply:
Still missing this.

Reply:
Done